### PR TITLE
Fix eslint ignitus-StudentSignUp

### LINKED ISSUE #54: Fix ESLINT Warnings
On running ```npm run lint``` ESLINT is showing 1081 problems (1075 errors, 6 warnings)
 621 errors and 0 warnings potentially fixable with the `--fix` option.
Fix as much as you can then feel free to send the PR.
We will be more than happy to merge it 👍 

screen shot 2018-10-07 at 11 05 29 am

### PR DESCRIPTION
**Pull request by darrylsepeda**
fix #54

Fix eslint ignitus-StudentSignUp

## Description
I splitted up my [previous pull request](https://github.com/Ignitus/Ignitus-Client-Side-Development/pull/57) to become smaller, hopefully can help the reviewer to easily merge this PR.

## Motivation and Context
Removing eslint errors from the existing source code

## How Has This Been Tested?
By running it locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by darrylsepeda** [comment]
okay will do tonight

File: src/ignitus-StudentSignUp/Components/Signup.js [hunk 1]
```diff
@@ -14,18 +14,15 @@ class Signup extends React.Component {
...
-  }
-
   render() {
+    const { value } = this.state;
     return (
       <div className="container _container-custom p-5">
         <div className="row shadow">
           <div className="col-md-6 p-0 container-image">
-            <img className="img-fluid _img-login d-block" src={loginImg} alt="login" />
+            <img className="img-fluid _img-login d-block" src={loginImg} alt="" />
```
Comment (meyfleur) [inline src/ignitus-StudentSignUp/Components/Signup.js]: please readd the alt tag again

File: src/ignitus-StudentSignUp/Components/Signup.js [hunk 2]
```diff
@@ -35,69 +32,68 @@ class Signup extends React.Component {
               </p>
             </div>
           </div>
+
           <div className="col-md-6  _container-form ">
             <div className="my-4">
-              <img className="img-fluid _img mx-auto d-block" src={logo} alt="logo" />
+              <img className="img-fluid _img mx-auto d-block" src={logo} alt="" />
             </div>
```
Comment (meyfleur) [inline src/ignitus-StudentSignUp/Components/Signup.js]: some for here alt tag should not be removed

File: src/ignitus-StudentSignUp/Components/Signup.js [hunk 3]
```diff
@@ -35,69 +32,68 @@ class Signup extends React.Component {
...
             <form>
```
Comment (meyfleur) [inline src/ignitus-StudentSignUp/Components/Signup.js]: lost overview here what exactly is the change in here?

File: src/ignitus-StudentSignUp/Components/Signup.js [hunk 4]
```diff
@@ -35,69 +32,68 @@ class Signup extends React.Component {
...
-                    id="pass"
-                    className="form-control"
-                    placeholder="Confirm Password"
-                    required
-                  />
-                </div>
-                <div className="text-center mb-3 mt-3">
-                  <button type="submit" className="btn btn-success btn-rounded px-3 py-2">
-                    Sign up
-                  </button>
```
Comment (meyfleur) [inline src/ignitus-StudentSignUp/Components/Signup.js]: lost overview in here what exactly was the change for here?

File: src/ignitus-StudentSignUp/Components/Signup.js (hunk 1 above)
Comment (darrylsepeda) [reply src/ignitus-StudentSignUp/Components/Signup.js]: I didn't realised its already updated, will update then. thanks

**Pull request closed without merging**